src/installer.ts

```diff
@@ -604,3 +605,16 @@ export function parseNodeVersionFile(contents: string): string {
 function isLatestSyntax(versionSpec): boolean {
   return ['current', 'latest', 'node'].includes(versionSpec);
 }
+
+export async function enableCorepack(input: string): Promise<void> {
+  let corepackArgs = ['enable'];
+  if (input.length > 0 && input !== 'false') {
+    if (input !== 'true') {
+      const packageManagers = input.split(' ');
+      corepackArgs.push(...packageManagers);
+    }
+    await exec.getExecOutput('corepack', corepackArgs, {
```

Comment:
please, use `getCommandOutput` or `getCommandOutputNotEmpty` instead of directly calling `exec.getExecOutput`